src/main/scala/com/wavesplatform/network/CoordinatorHandler.scala

```diff
@@ -83,18 +83,18 @@ class CoordinatorHandler(checkpointService: CheckpointService,
       Signed.validateSignatures(b).flatMap(b => processBlock(b, false))
     } map {
       case Right(None) =>
-        log.trace(s"Block ${b.uniqueId} already appended")
+        log.trace(s"Block $b already appended")
       case Right(Some(newScore)) =>
-        log.debug(s"Appended block ${b.uniqueId}")
+        log.debug(s"Appended block $b")
         if (b.transactionData.isEmpty)
           allChannels.broadcast(BlockForged(b), Some(ctx.channel()))
         miner.scheduleMining()
         allChannels.broadcast(LocalScoreChanged(newScore))
       case Left(is: InvalidSignature) =>
-        peerDatabase.blacklistAndClose(ctx.channel(), s"Could not append block ${b.uniqueId}: $is")
+        peerDatabase.blacklistAndClose(ctx.channel(), s"Could not append block $b: $is")
       case Left(ve) =>
         BlockStats.declined(b, BlockStats.Source.Broadcast)
-        log.debug(s"Could not append block ${b.uniqueId}: $ve")
+        log.debug(s"Could not append block $b: $ve")
```

Comment:
Trimmed ID is not useful in scenario when you copy-paste block's ID from log and lookup for it in the explorer
Should we change back to full ID, or simply output full ID in toString?